assets/babel-plugin.js

```diff
@@ -3,8 +3,12 @@ const path = require('path');
 
 function readImportMapFile(explicitPath, dir) {
   if (explicitPath) {
-    const explicitImportMap = path.join(process.cwd(), dir, explicitPath);
-    return fs.readFileSync(explicitImportMap, {encoding: 'utf8'});
+    if(path.isAbsolute(explicitPath)){
+      return fs.readFileSync(explicitPath, {encoding: 'utf8'});
+    } else {
+      const explicitImportMap = path.join(process.cwd(), dir, explicitPath);
+      return fs.readFileSync(explicitImportMap, {encoding: 'utf8'});
+    }
```

Comment:
complete nit: move from if-else to just if

```suggestion
    if(path.isAbsolute(explicitPath)){
      return fs.readFileSync(explicitPath, {encoding: 'utf8'});
    }
    const explicitImportMap = path.join(process.cwd(), dir, explicitPath);
    return fs.readFileSync(explicitImportMap, {encoding: 'utf8'});
```